shiny/render/_dataframe.py

```diff
@@ -270,6 +279,15 @@ async def transform(self, value: DataFrameResult) -> Jsonifiable:
             )
         return value.to_payload()
 
+    def input_selected_rows(self) -> Optional[tuple[int]]:
```

Comment:
I think it would be better if this always returned a tuple. Especially when the selection mode is multiple, having zero rows selected is a different set of selected rows (it's a selection of zero rows), but it is not a fundamentally different kind (like `None`).

Returning an empty tuple would be similar to `brushed_points`, which, if no points are selected, returns an empty data frame instead of `None`.

Reply:
Oh, actually, after testing it out, I see that:
- If the mode is single or multiple and no rows are selected, this returns an empty tuple.
- If the mode is none, then this returns `None`.

It would help if the docs accurately reflected this.

Also, a small nit: 

```suggestion
    def input_selected_rows(self) -> tuple[int] | None:
```

Although this is technically equivalent to using `Optional`, we've tried to use `Optional` to convey that an argument is optional, which is different from this case.